src/NosnocProblemOptions.m

```diff
@@ -53,20 +53,15 @@
         dcs_mode DcsMode = DcsMode.Stewart
 
         % lift complementarities
-        lift_complementarities(1,1) logical = 1
-        lower_bound_comp_lift(1,1) logical = 1
+        lift_complementarities(1,1) logical = 0
+        lower_bound_comp_lift(1,1) logical = 0
 
         % TODO: Initialization is really a solver concern, but there is not really a good extraction approach yet.
         % Initialization - Stewart
         lp_initialization(1,1) logical = 0
-        initial_theta(1,1) double {mustBeReal, mustBeFinite} = 1
-        initial_lambda(1,1) double {mustBeReal, mustBeFinite} = 1
-        initial_mu(1,1) double {mustBeReal, mustBeFinite} = 1
 
         % Initialization - Step
         initial_alpha(1,1) double {mustBeReal, mustBeFinite} = 1
-        initial_lambda_0(1,1) double {mustBeReal, mustBeFinite} = 1
-        initial_lambda_1(1,1) double {mustBeReal, mustBeFinite} = 1
         initial_beta(1,1) double {mustBeReal, mustBeFinite} = 1
```

Comment:
Here I am not sure how to get rid of it, maybe add a todo, we can discuss with armin.